Formula/p/pivit.rb

```diff
@@ -0,0 +1,26 @@
+class Pivit < Formula
+  desc "Sign and verify data using hardware (Yubikey) backed x509 certificates (PIV)"
+  homepage "https://github.com/cashapp/pivit"
+  url "https://github.com/cashapp/pivit/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "67a2f353c820aad9c9a4cc937ba256c75cb36044baa0578b74a1bfbdf4e6bd90"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  on_linux do
+    depends_on "pcsc-lite"
+  end
+
+  def install
+    system "make", "pivit"
+    bin.install "pivit"
+  end
+
+  test do
+    output = pipe_output("#{bin}/pivit -p 2>&1").strip
+    pcsc_not_running = "the Smart card resource manager is not running"
+    card_not_found = "no smart card found with certificate in slot 9e"
+    certificate_prefix = "-----BEGIN CERTIFICATE-----"
+    assert_match(/(#{pcsc_not_running}|#{card_not_found}|#{certificate_prefix})/, output)
```

Comment:
Let's just optimise for the case in CI

Reply:
For CI the expected result would be the `pcscd` service not running (since it's a virtual machine with no smart card attached to it).
Optimizing for that case would probably cause failures if these tests run manually on a real machine.

Is it reasonable to assume that these tests are meant mostly for CI?

-----------
For context about the `pcscd` service - this is the service that's responsible for communicating with smart cards hardware (like Yubikeys). In Mac, it is not started by default. Instead, it'll prompt the user to start after the hardware is attached via USB.
That's reason there are 3 possible outputs in this test:
1. For CI, the service is obviously not running. The command will fail with exit status `1`, and the exception will be printed to `stderr`.
2. In a real Mac machine:
    * If `pcscd` isn't running we can expect the same result for CI
    * Otherwise, we can expect either a message specifying the card/certificate isn't found, or a certificate being printed out.

Reply:
These tests are only ever run when testing the formula, so I don't think there is much of a need to optimise for users who have a full setup since they will know where their system differs from the test.

For CI however I think it's better to keep the test as simple as possible

Reply:
Got it! That makes the test much simpler :)